Assets/Python/Victory.py

```diff
@@ -532,21 +532,21 @@ def checkTurn(iGameTurn, iPlayer):
 				
 		# third goal: acquire 4000 gold by trade by 1200 AD
 		if isPossible(iTamils, 2):
-			iTradeGold = 0
+			iTradeGold = 0.0

```

Comment:
Instead of a float, this should still store an integer but instead by a factor of 100, Civ4 does not need more accuracy.

Reply:
Only iTradeGold of all instances of fTamilTradeGold?

Reply:
All instances where floats are used to store the accrued Tamil gold.

Reply:
Done.

I also included Italy to the list of conditional spawns and moved the code inside the check of line 1129.